.github/workflows/dev-checks.yml

```diff
@@ -82,15 +97,24 @@ jobs:
           echo "TARGET_BRANCH=$(echo ${GITHUB_BASE_REF} | tr / -)" >> $GITHUB_ENV
           echo "GITHUB_TOKEN=${{ secrets.CIPIPELINE_NEXUS_PASSWORD  }}" >> $GITHUB_ENV
         
-      - name: Check branch name
+      - name: Check branch and PR title format
```

Comment:
When we are moving it to python, why do we need it the check here ?

Also in the `tessell-harness-library` code, include `revert-`

Reply:
That's an old code I wrote initially just for checking the format purpose not for extraction, we can remove it.